src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/email/AppellantRequestHearingRequirementsPersonalisationEmail.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.appellant.email;
+
+import static java.util.Objects.requireNonNull;
+
+import com.google.common.collect.ImmutableMap;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Map;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.*;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.service.DirectionFinder;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.service.RecipientsFinder;
+import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.CustomerServicesProvider;
+
+@Service
+public class AppellantRequestHearingRequirementsPersonalisationEmail implements EmailNotificationPersonalisation {
+    private final String appellantRequestHearingRequirementsTemplateId;
+    private final String iaAipFrontendUrl;
+    private final RecipientsFinder recipientsFinder;
+    private final DirectionFinder directionFinder;
+    private final CustomerServicesProvider customerServicesProvider;
+
+    public AppellantRequestHearingRequirementsPersonalisationEmail(
+        @Value("${govnotify.template.requestHearingRequirements.appellant.email}") String appellantRequestHearingRequirementsTemplateId,
+        @Value("${iaAipFrontendUrl}") String iaAipFrontendUrl,
+        RecipientsFinder recipientsFinder,
+        DirectionFinder directionFinder,
+        CustomerServicesProvider customerServicesProvider
+    ) {
+        this.appellantRequestHearingRequirementsTemplateId = appellantRequestHearingRequirementsTemplateId;
+        this.iaAipFrontendUrl = iaAipFrontendUrl;
+        this.recipientsFinder = recipientsFinder;
+        this.directionFinder = directionFinder;
+        this.customerServicesProvider = customerServicesProvider;
+    }
+
+    @Override
+    public String getTemplateId() {
+        return appellantRequestHearingRequirementsTemplateId;
+    }
+
+    @Override
+    public Set<String> getRecipientsList(final AsylumCase asylumCase) {
+        return recipientsFinder.findAll(asylumCase, NotificationType.EMAIL);
+    }
+
+    @Override
+    public String getReferenceId(Long caseId) {
+        return caseId + "_REQUEST_HEARING_REQUIREMENTS_DIRECTION_AIP_EMAIL";
+    }
+
+    @Override
+    public Map<String, String> getPersonalisation(AsylumCase asylumCase) {
+        requireNonNull(asylumCase, "asylumCase must not be null");
+
+        final Direction direction =
+            directionFinder
+                .findFirst(asylumCase, DirectionTag.LEGAL_REPRESENTATIVE_HEARING_REQUIREMENTS)
```

Comment:
@srinivasuluputcha we should look to introduce a new DirectionTag here: **APPELLANT_HEARING_REQUIREMENTS**.

This direction going to the appellant, not going to the legal representative - @jkumar-moj would you agree?